app/Presenters/ActionlogPresenter.php

```diff
@@ -38,22 +38,63 @@ public function item()
 
     public function icon()
     {
-        $itemicon = 'fas fa-paperclip';
-
-        if ($this->itemType() == 'asset') {
-            return 'fas fa-barcode';
-        } elseif ($this->itemType() == 'accessory') {
-            return 'far fa-keyboard';
-        } elseif ($this->itemType() == 'consumable') {
-            return 'fas fa-tint';
-        } elseif ($this->itemType() == 'license') {
-            return 'far fa-save';
-        } elseif ($this->itemType() == 'component') {
-            return 'far fa-hdd';
-        } elseif ($this->itemType() == 'user') {
-            return 'fa-solid fa-people-arrows';
+        
+        // User related icons
+        if ($this->itemType() == 'user') {
```

Comment:
Wouldn't this be a little nicer with a `switch()....case` kind of deal? Not a hill I'm gonna die on by any means.